src/mds/FSMap.cc

```diff
@@ -133,6 +133,7 @@ void Filesystem::dump(Formatter *f) const
 void FSMap::dump(Formatter *f) const
 {
   f->dump_int("epoch", epoch);
+  f->dump_string("btime", fmt::format("{}", btime));
```

Comment:
There is a related build failure. See: https://jenkins.ceph.com/job/ceph-dev-new-build/ARCH=x86_64,AVAILABLE_ARCH=x86_64,AVAILABLE_DIST=centos9,DIST=centos9,MACHINE_SIZE=gigantic/85781//consoleFull

```
/home/jenkins-build/build/workspace/ceph-dev-new-build/ARCH/x86_64/AVAILABLE_ARCH/x86_64/AVAILABLE_DIST/centos9/DIST/centos9/MACHINE_SIZE/gigantic/release/18.2.4-1376-g09858a4b/rpm/el9/BUILD/ceph-18.2.4-1376-g09858a4b/src/mds/FSMap.cc:136:38:   required from here
/usr/include/fmt/core.h:1729:7: error: static assertion failed: Cannot format an argument. To make type T formattable provide a formatter<T> specialization: https://fmt.dev/latest/api.html#udt
 1729 |       formattable,
      |       ^~~~~~~~~~~
```